crawl_sourcecode.py

```diff
@@ -67,3 +68,9 @@ def print_table(table, indent=0):
 
 table = symtable.symtable(source, 'a', 'exec')
 print_table(table)
+
+print()
+print('======== dis.dis ========')
+print()
+co = compile(source, filename, 'exec')
```

Comment:
`dis.dis` returns `None`, it prints the disassembly.

Reply:
Allright, will change it later on. This is sort of a scratch script to check for scope detection working properly.